torch/utils/serialization/read_lua_file.py

```diff
@@ -160,7 +160,11 @@ def read_storage(reader, version):
         # source:
         # https://github.com/torch/torch7/blob/master/generic/Storage.c#L244
         size = reader.read_long() * element_size
-        return python_class.from_buffer(reader.f.read(size), 'native')
+        # because there's no torch.cuda.FloatStorage.from_buffer
+        if python_class == torch.cuda.FloatStorage:
+            return torch.FloatStorage.from_buffer(reader.f.read(size), 'native').cuda()
+        else:
+            return python_class.from_buffer(reader.f.read(size), 'native')
     return read_storage
 
 
```

Comment:
I think we need to call some decorators for cudnn modules as well, like:

`ensure_attr('cudnn.SpatialConvolution', 'bias', 'finput', 'fgradInput', 'gradWeight', 'gradBias')
ensure_attr('cudnn.SpatialBatchNormalization', 'weight', 'bias', 'gradWeight', 'gradBias')
ensure_attr('cudnn.BatchNormalization', 'weight', 'bias', 'gradWeight', 'gradBias')`